Lib/test/support/interpreters/channels.py

```diff
@@ -6,8 +6,8 @@
 
 # aliases:
 from _interpchannels import (
-    ChannelError, ChannelNotFoundError, ChannelClosedError,
-    ChannelEmptyError, ChannelNotEmptyError,
+    ChannelError, ChannelNotFoundError, ChannelClosedError,  # noqa: F401
+    ChannelEmptyError, ChannelNotEmptyError,  # noqa: F401
```

Comment:
In this specific case maybe better to add a per-file ignore?

Reply:
Or would you prefer the two unused imports on the same line? That way we still check the rest of the file.

```suggestion
from _interpchannels import (
    ChannelError, ChannelNotFoundError, ChannelEmptyError,
    ChannelClosedError, ChannelNotEmptyError,  # noqa: F401
)
```

Reply:
I'm fine with the current PR: two lines with `# noqa: F401`.